invenio_communities/assets/semantic-ui/js/invenio_communities/requests/requests.js

```diff
@@ -302,9 +302,9 @@ export const RequestsResults = ({
   );
 };
 
-export const RequestsResultsGridItemTemplate = ({ result }) => {
+export const RequestsResultsGridItemTemplate = ({ result, community }) => {
   return (
-    <Card fluid href={`/me/requests/${result.metadata.id}`}>
+    <Card fluid href={`/communities/${community.id}/requests/${result.metadata.id}`}>
```

Comment:
Thank you for the fix. If we manage in the future to inject this from the service, it would be better.
We might want to customize this URL for CERN instances.